x/emissions/proto/emissions/v5/genesis.proto

```diff
@@ -0,0 +1,388 @@
+import "emissions/v3/nonce.proto";
+import "emissions/v3/params.proto";
+import "emissions/v3/reputer.proto";
+import "emissions/v3/score.proto";
+import "emissions/v3/stake.proto";
+import "emissions/v3/topic.proto";
+import "emissions/v3/types.proto";
+import "emissions/v3/worker.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/allora-network/allora-chain/x/emissions/types";
+
+// GenesisState is the state that must be provided at genesis.
+message GenesisState {
+  reserved 13, 14, 15;
+  reserved "latestInfererScoresByWorker", "latestForecasterScoresByWorker", "latestReputerScoresByReputer";
+
+  // params defines all the parameters of the module.
+  emissions.v3.Params params = 1 [(gogoproto.nullable) = false];
+
+  /// TOPIC
+  // the next topic id to be used, equal to the number of topics that have been
+  // created
+  uint64 next_topic_id = 3;
+  // every topic that has been created indexed by their topicId starting from 1
+  // (0 is reserved for the root network)
+  repeated TopicIdAndTopic topics = 4;
+  repeated uint64 active_topics = 5;
+  // every topic that has been churned and ready to be rewarded i.e. reputer
+  // losses have been committed
+  repeated uint64 rewardable_topics = 6;
+  // for a topic, what is every worker node that has registered to it?
+  repeated TopicAndActorId topic_workers = 7;
+  // for a topic, what is every reputer node that has registered to it?
+  repeated TopicAndActorId topic_reputers = 8;
+  // map of (topic) -> nonce/block height
+  repeated TopicIdAndBlockHeight topic_reward_nonce = 9;
+
+  /// SCORES
+  // map of (topic, block_height, worker) -> score
+  repeated TopicIdBlockHeightScores inferer_scores_by_block = 10;
+  // map of (topic, block_height, worker) -> score
+  repeated TopicIdBlockHeightScores forecaster_scores_by_block = 11;
+  // map of (topic, block_height, reputer) -> score
+  repeated TopicIdBlockHeightScores reputer_scores_by_block = 12;
+  // map of (topic, block_height, worker) -> score
+  repeated TopicIdActorIdScore inferer_score_emas = 60;
+  // map of (topic, block_height, worker) -> score
+  repeated TopicIdActorIdScore forecaster_score_emas = 61;
+  // map of (topic, block_height, reputer) -> score
+  repeated TopicIdActorIdScore reputer_score_emas = 62;
+  // map of (topic, reputer) -> listening coefficient
+  repeated TopicIdActorIdListeningCoefficient reputer_listening_coefficient = 16;
+  // map of (topic, reputer) -> previous reward (used for EMA)
+  repeated TopicIdActorIdDec previous_reputer_reward_fraction = 17;
+  // map of (topic, worker) -> previous reward for inference (used for EMA)
+  repeated TopicIdActorIdDec previous_inference_reward_fraction = 18;
+  // map of (topic, worker) -> previous reward for forecast (used for EMA)
+  repeated TopicIdActorIdDec previous_forecast_reward_fraction = 19;
+  // map of (topic, forecaster) -> ratio of forecaster score
+  repeated TopicIdAndDec previous_forecaster_score_ratio = 20;
```

Comment:
should close PROTO-2349, but I don't see any corresponding change in genesis.go, did we end up fixing this case?

Reply:
hmm i did update both of the `genesis.go` files...